src/CommunityVoices/Public/js/batch-upload-confirm.js

```diff
@@ -22,7 +22,14 @@ function postData(form) {
         $.ajax({
           url : $("#actualForm").attr('action'),
           type: $("#actualForm").attr('method'),
-          data: data
+          data: data,
+          success: function () {
+            // This goes back to /quotes because we are at /quotes/confirm
+            window.location.href = '.';
+          },
+          error: function () {
+            alert('Could not upload all quotes. Please contact dashboard@oberlin.edu');
```

Comment:
@Sammidysam In the jQuery I believe I am checking when quotes have errors that will prevent their submission so this seems a little redundant. Thus, a quote/source upload that would lead to a failed request should be anticipated ahead of time. I know that hypothetically it is possible for an issue to come up that isn't anticipated (i.e. database goes down, server issues, etc.) but all errors related to the wrong fields being specified should be caught. Do you think it's worth it specifying an additional alert if sources/quotes bypass that first layer of checking, and still can't be uploaded? I almost think at that point it is more worth notifying you or whoever is in charge of dev, because that would indicate that something is seriously wrong with the system that wasn't anticipated.

Reply:
I definitely understand not having the *need* to specify a function for this error, but we have historically had a lot of issues with server downtime with Community Hub since we run on as little resources as possible until we actually have income.  So, the effort of writing 3 lines just to alert for a catastrophic error is really worthwhile.  I am not certain we have a mailer set up with Community Voices, or otherwise this would be an error that sends an email to the development team.

Reply:
@Sammidysam Ahh gotcha. Yeah mailer is where my mind went too for this error since it indicates something "catastrophic" as you said. But if we don't have one then this makes sense too.